### PR TITLE
Design: Add a transaction page

### PR DESCRIPTION
**Pull request by 0xbaggi**
Done :
- Created CircularMenuItemWrap to add custom Text from CircularMenuItem
- Dynamic button
- Top Icon and Title

To-do:
- Set right icons

### CODE REVIEW

File: pubspec.lock [hunk 1]
```diff
@@ -85,13 +92,6 @@ packages:
       url: "https://pub.dartlang.org"
     source: hosted
     version: "1.0.5"
-  equatable:
```
Comment (FedericoBruzzone) [inline pubspec.lock:88]: Why did you remove this?

File: pubspec.lock [hunk 2]
```diff
@@ -113,13 +113,6 @@ packages:
       url: "https://pub.dartlang.org"
     source: hosted
     version: "6.1.4"
-  fl_chart:
```
Comment (FedericoBruzzone) [inline pubspec.lock:116]: Why did you remove this?

**Comment by FedericoBruzzone** [comment]
Quick question.
Can you send us a screen of your work?

File: lib/pages/structure.dart [hunk 3]
```diff
@@ -28,84 +27,84 @@ class _StructureState extends State<Structure> {
...
+      ),
+      body: Center(
+        child: _pages.elementAt(_selectedIndex),
+      ),
+      bottomNavigationBar: BottomNavigationBar(
+        type: BottomNavigationBarType.fixed,
+        selectedItemColor: Colors.white,
+        backgroundColor: const Color.fromRGBO(204, 204, 204, 1),
+        currentIndex: _selectedIndex,
+        //showSelectedLabels: false,
```
Comment (FedericoBruzzone) [inline lib/pages/structure.dart:73]: Remove this

File: lib/pages/structure.dart [hunk 4]
```diff
@@ -28,84 +27,84 @@ class _StructureState extends State<Structure> {
...
+        //showUnselectedLabels: false,
```
Comment (FedericoBruzzone) [inline lib/pages/structure.dart:74]: Same above

**Review by FedericoBruzzone — CHANGES_REQUESTED** [review]

File: lib/model/CircularMenuItemWrap.dart [hunk 5]
```diff
@@ -0,0 +1,32 @@
+import 'package:circular_menu/circular_menu.dart';
+import 'package:flutter/material.dart';
+
+class CircularMenuItemWrap extends CircularMenuItem {
```
Comment (FedericoBruzzone) [inline lib/model/CircularMenuItemWrap.dart:4]: You can find the `widget test file` in a folder `~./test/widget` (something like this). 
Please, can you add the test of this widget?

**Review by FedericoBruzzone — CHANGES_REQUESTED** [review]

File: pubspec.lock (hunk 2 above)
Comment (0xbaggi) [reply pubspec.lock:116]: It is an unused library.

**Comment by 0xbaggi** [comment]
> 



> You wrote in you PR comment `Set right icons`. What it means? Should we open an issue about it?

I mean we have to choose the most appropriate icons on the page, for now I have only selected a few which I think will fit, but we can find better choices. I think we should open a issue when we have coded a first version of all the app because this task is needed for all pages.

**Comment by FedericoBruzzone** [comment]
> 
 
You should be able to open the issue about it, right?
Thanks for making it known!

**Comment by FedericoBruzzone** [comment]
> > Quick question. Can you send us a screen of your work?
> 
> Immagine 2022-11-10 153555
> 
> From Pixel XL (API 31).

Looks good!

**Comment by 0xbaggi** [comment]
I have implemented the other changes in the last commit of this PR.

**Comment by FedericoBruzzone** [comment]
Can I merge this PR?
@0xbaggi

**Comment by 0xbaggi** [comment]
> After your opening the issue can I merge this PR? @0xbaggi

Yes! I just opened the issue ( #32 )

**Pull request merged**